student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
```

Comment:
- [ ] Kindly import the Person class in this file to allow inheritance.

  ```
  require_relative './person.rb'
  ```

Reply:
> * [ ]  Kindly import the Person class in this file to allow inheritance.
>   ```
>   require_relative './person.rb'
>   ```

- [ ] As the previous reviewer suggested, kindly add the mentioned line on the `student.rb` file. :+1: